content/components/action-bar.mdx

```diff
@@ -0,0 +1,277 @@
+  </Label>
+  <Label
+    as="a"
+    href=""
+    size="large"
+    variant="secondary"
+    sx={{textDecoration: 'none'}}
+  >
+    <img
+      width="16"
+      height="16"
+      alt=""
+      src="https://user-images.githubusercontent.com/293280/123878720-80d51480-d8f4-11eb-9b10-d02a1cb606f8.png"
+      style="vertical-align: middle; margin-right: 4px;"
+    />
+    Figma
+  </Label>
+  <Label
+    as="a"
+    href="https://primer.style/view-components/components/alpha/actionbar"
+    size="large"
+    variant="secondary"
+    sx={{textDecoration: 'none'}}
+  >
+    <img
+      width="16"
+      height="16"
+      alt=""
+      src="https://user-images.githubusercontent.com/18661030/171468311-184bfd70-128e-4bfc-b482-96d0de49d42a.png"
+      style="vertical-align: middle; margin-right: 4px;"
+    />
+    Rails
+  </Label>
+</LabelGroup>
+-->
+
+## Overview
+
+Use an action bar to render multiple icon buttons in a row. Buttons can be split into groups by adding a divider. When there is not enough space, buttons that don't fit will be added to an overflow menu.
+
+## Anatomy
+
+<img width="960" alt="A diagram of an action bar with a few buttons, a divider and at the end a button that opens an overflow menu." src="https://user-images.githubusercontent.com/378023/188037013-e0c071ee-19f5-4a1c-9deb-3bc695d71cec.png" />
+
+## Content
+
+### Buttons
+
+An action bar should only contain icon buttons with the `invisible` variant (no border/background).
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/378023/187869924-a28f1932-f2ac-41a2-8191-8ae7ac7b4c96.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Use only invisible icon buttons in an action bar.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/378023/187869945-9ad1403b-5803-4f74-8059-ca2383b17c48.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Don't use other variants or components in an action bar.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Dividers
+
+Dividers can be added to visually group related buttons.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/378023/187867904-7790eed2-35bd-4434-9fbe-89778a69d13f.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Use a divider between buttons.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/378023/187869152-6f6fb191-bdf3-4e77-9ad5-53685190f5a3.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Don't use a divider at the beginning or end of the action bar.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Overflow menu
+
+When the buttons don't fit in the available space, an overflow button ("kebab" icon) is added at the end of the action bar. Signaling that there are more actions available. Clicking on the overflow button opens a menu with the remaining actions that didn't fit.
```

Comment:
Would it make sense to explain what is the order by which the elements in the list are sorted?

I can think of the [Stack data type](https://en.wikipedia.org/wiki/Stack_(abstract_data_type)) and how they visually describe the push action:

<img width="825" alt="Screenshot 2022-09-06 at 08 28 16" src="https://user-images.githubusercontent.com/912236/188562933-7722067f-585d-4fd0-93fc-aedea5f25761.png">

Reply:
That's a great idea. Added this: 

<img width="960" alt="Full (2)" src="https://user-images.githubusercontent.com/378023/188835345-0cfd3376-1658-496f-a78b-f5977aa2198c.png">

👀  [Preview](https://primer-04c6110efb-26441320.drafts.github.io/components/action-bar#sorting)

Reply:
Love it ❤️